Localization/nl-NL/about.po

```diff
@@ -0,0 +1,33 @@
+msgid ""
+msgstr ""
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/about/\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20200705\n"
+"Target-Language: nl-NL\n"
+"Target-Version: \n"
```

Comment:
Please put a value here. Either just `1.0` or you start versioning also your local updates and use `1.0.0`. (The latter would allow you to commit updates that you make to your translation by bumping up the last number. It's not mandatory though.)